### ISSUE TITLE
[Feature] Stat for First added date/time

### ISSUE THREAD

**Issue by hontheinternet** [stashapp/stash#3943]
I will implement it when I have time. 
Creating this issue as a placeholder for when I implement it. :)

**Comment by MrX292** [comment]
do you mean `Created At:`?

**Comment by hontheinternet** [comment]
> do you mean `Created At:`?

No, like a date/time since when you started collecting your stash. It is good complement to the statistics in the Stats page.

**Comment by LeGrosFromage** [comment]
This could be quite useful.  I'm really anal about having a date show on the scene card, which for older scenes (I have some from 2000) can be really hard to track down.  I've had to resort to file encoding date to try and be able to put something in.  

Will this secondary/extra date show on the scene card?
